tests/saitests/py3/sai_qos_tests.py

```diff
@@ -3111,7 +3125,8 @@ def runTest(self):
             assert (recv_counters[pg] == recv_counters_base[pg])
             # recv port no ingress drop
             for cntr in ingress_counters:
-                assert (recv_counters[cntr] == recv_counters_base[cntr])
+                if platform_asic and platform_asic == "broadcom-dnx" and cntr == 1:
+                    assert(recv_counters[cntr] == recv_counters_base[cntr])
```

Comment:
what is the purpose of this change ? Can we have this applicable to chassis only ?

Reply:
Vineet, If i remember correctly the packet drops are counted as part of queue drop stats and not port drop stats , So the In Discards counter was not working and we had to bypass that  counter check for dnx